docs/version-history/introduction-version-history.rst

```diff
@@ -2,67 +2,98 @@ Introduction to HTCondor Versions
-specific release series, and all the HTCondor versions found in that
-series.
+different releases are. The rest of the sections each describe the
+specific releases.
 
 HTCondor Version Number Scheme
 ------------------------------
 
-HTCondor's versioning scheme reflects the fact that it
-is both a production system and a research project. The numbering scheme
-was primarily taken from the Linux kernel's version numbering, so if you
-are familiar with that, it should seem quite natural.
-
-There will usually be two HTCondor versions available at any given time,
-the stable version, and the development version. All versions of HTCondor 
-now have exactly three numbers, separated by "."
-
--  The first number represents the major version number, and will change
-   very infrequently.
--  The thing that determines whether a version of HTCondor is stable or
-   development is the second digit. Even numbers represent stable
-   versions, while odd numbers represent development versions.
--  The final digit represents the minor version number, which defines a
-   particular version in a given release series.
-
-The Stable Release Series
--------------------------
-
-People expecting the stable, production HTCondor system should download
-the stable version, denoted with an even number in the second digit of
-the version string. 
-
-On the stable series, new minor version releases will only be made for
-bug fixes and to support new platforms. No new features will be added to
-the stable series. People are encouraged to install new stable versions
-of HTCondor when they appear, since they probably fix bugs you care
-about. Hopefully, there will not be many minor version releases for any
-given stable series.
-
-All minor release version within a stable series are wire-compatible
-with each other, so if an administrator needs to upgrade the central
-manager one day, the pool will run with an older scheduler or execute
-nodes.
-
-The Development Release Series
-------------------------------
+Starting with HTCondor 9.2.0, the version number scheme changed.
+We made this change to give us additional flexibility in releasing
+small updates to address specific issues, without disturbing the normal
+development of HTCondor.  The version number will still retain the
+MAJOR.MINOR.PATCH form with slightly different meanings.  We have borrowed
+ideas from `Semantic Versioning <https://semver.org/>`_.
+
+-   The MAJOR number increments for each new Long Term Support (LTS) release.
+    A new LTS release may have backward-incompatible changes and may require
+    updates to configuration files.  If the current LTS release is 9.0.6.
+    The next one will be 10.0.0.  A new LTS release is expected every twelve
+    to eighteen months.
+
+-   The MINOR number increments during the development of new features. This
+    number stays at 0 for LTS releases. The current release is 9.2.0.
+    A regular release is expected to increment every one to two months.
```

Comment:
This is rather stealthily adding a definition of a type of release.  Perhaps start this section off as "The MINOR number increments for each regular release, which we make during the development of new features."

It may be better to introduce the four kinds of releases (major, LTS patch, feature/regular, feature/regular patch) before specifying the individual version number components.